README_zh-CN.md

```diff
@@ -290,33 +316,32 @@ MMAction2 将跟进学界的最新进展，并支持更多算法和框架。如
-- [MMDetection3D](https://github.com/open-mmlab/mmdetection3d): OpenMMLab 新一代通用 3D 目标检测平台
-- [MMRotate](https://github.com/open-mmlab/mmrotate): OpenMMLab 旋转框检测工具箱与测试基准
-- [MMTracking](https://github.com/open-mmlab/mmtracking): OpenMMLab 一体化视频目标感知平台
-- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation): OpenMMLab 语义分割工具箱
-- [MMOCR](https://github.com/open-mmlab/mmocr): OpenMMLab 全流程文字检测识别理解工具包
-- [MMPose](https://github.com/open-mmlab/mmpose): OpenMMLab 姿态估计工具箱
-- [MMHuman3D](https://github.com/open-mmlab/mmhuman3d): OpenMMLab 人体参数化模型工具箱与测试基准
-- [MMFewShot](https://github.com/open-mmlab/mmfewshot): OpenMMLab 少样本学习工具箱与测试基准
-- [MMAction2](https://github.com/open-mmlab/mmaction2): OpenMMLab 新一代视频理解工具箱
-- [MMFlow](https://github.com/open-mmlab/mmflow): OpenMMLab 光流估计工具箱与测试基准
-- [MMDeploy](https://github.com/open-mmlab/mmdeploy): OpenMMLab 模型部署框架
-- [MMRazor](https://github.com/open-mmlab/mmrazor): OpenMMLab 模型压缩工具箱与测试基准
-- [MIM](https://github.com/open-mmlab/mim): OpenMMlab 项目、算法、模型的统一入口
-- [Playground](https://github.com/open-mmlab/playground): 收集和展示 OpenMMLab 相关的前沿、有趣的社区项目
+## 🙌 参与贡献 [🔝](#-table-of-contents)
+
+我们感谢所有参与改进MMAction2的贡献，更多贡献指南请参阅MMCV下的[CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/2.x/CONTRIBUTING.md)
+
+## 🤝 致谢 [🔝](#-table-of-contents)
+
+MMAction2是一个来自众多高校和企业的研究人员及工程师共同参与构建的开源项目，我们感谢所有实现他们的算法，添加新功能的贡献者，以及那些提供有价值的反馈的用户。我们希望通过提供一款灵活的能够实现现有算法以及开发新的模型工具包，来服务于日益增长的研究群体，
+
+## 🏗️ OpenMMLab所有项目 [🔝](#-table-of-contents)
+
+- [MMEngine](https://github.com/open-mmlab/mmengine): OpenMMLab训练深度学习模型的基础库
+- [MMCV](https://github.com/open-mmlab/mmcv): OpenMMLab计算机视觉基础库
+- [MMPreTrain](https://github.com/open-mmlab/mmpretrain): OpenMMLab预训练工具包及基准
+- [MMagic](https://github.com/open-mmlab/mmagic): Open**MM**Lab高级（**A**）生成式（**G**）及智能（**I**）创建（**C**）工具包
+- [MMDetection](https://github.com/open-mmlab/mmdetection): OpenMMLab物体检测工具包及基准
+- [MMDetection3D](https://github.com/open-mmlab/mmdetection3d): OpenMMLab下一代通用3D物体检测平台
+- [MMRotate](https://github.com/open-mmlab/mmrotate): OpenMMLab旋转物体检测工具包及基准
+- [MMTracking](https://github.com/open-mmlab/mmtracking): OpenMMLab视频感知工具包及基准
+- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation): OpenMMLab语义分割工具包及基准
+- [MMOCR](https://github.com/open-mmlab/mmocr): OpenMMLab文本检测，识别及理解工具包
+- [MMPose](https://github.com/open-mmlab/mmpose): OpenMMLab姿态预测工具包及基准
+- [MMHuman3D](https://github.com/open-mmlab/mmhuman3d): OpenMMLab 3D人物参数模型工具包及基准
+- [MMFewShot](https://github.com/open-mmlab/mmfewshot): OpenMMLab少样本学习工具包及基准
+- [MMAction2](https://github.com/open-mmlab/mmaction2): OpenMMLab下一代动作理解工具并及基准
+- [MMFlow](https://github.com/open-mmlab/mmflow): OpenMMLab光流法工具并及基准
+- [MMDeploy](https://github.com/open-mmlab/mmdeploy): OpenMMLab模型部署框架
+- [MMRazor](https://github.com/open-mmlab/mmrazor): OpenMMLab模型压缩工具包及基准
+- [MIM](https://github.com/open-mmlab/mim): MIM用于安装OpenMMLab所需包
+- [Playground](https://github.com/open-mmlab/playground): 一个用于收集和展示基于OpenMMLab构建的优质项目中心站点
```

Comment:
These descriptions are used in all repos in a unified form. So we prefer not to change them in MMAction2 if there is no special reason.

Reply:
Okay, I restore them to the original ones